### PR TITLE
Implement key and key ID tags

### PR DESCRIPTION
**Pull request by olorin**
Implement spec changes in #62, adding tags to key and key ID for easy identification and grepping purposes.

@erikd-ambiata @thumphries

### CODE REVIEW

File: zodiac-tsrp/src/Zodiac/TSRP/Data/Key.hs [hunk 1]
```diff
@@ -20,33 +24,73 @@ import           GHC.Generics (Generic)
...
 
+import           Tinfoil.Data.Key (SymmetricKey(..))
+import           Tinfoil.Data.Key (parseSymmetricKey, renderSymmetricKey)
 import           Tinfoil.Encode (hexEncode)
 
--- | Identifier for either a symmetric or asymmetric key. Should be
--- globally unique. Sixteen bytes long.
+-- | Identifier for a TSRP key. Should be globally unique. Sixteen bytes long
+-- plus six byte for the tag prefix.
 newtype KeyId =
```
Comment (erikd-ambiata) [inline zodiac-tsrp/src/Zodiac/TSRP/Data/Key.hs]: "six bytes" with an 's'?

**Comment by erikd-ambiata** [comment]
:100: with comment.

File: zodiac-tsrp/src/Zodiac/TSRP/Data/Key.hs (hunk 1 above)
Comment (olorin) [inline zodiac-tsrp/src/Zodiac/TSRP/Data/Key.hs]: Indeed, thanks.

**Comment by olorin** [comment]
@erikd-ambiata would you mind taking a look over #62 before I merge this? Would prefer the spec change goes in before the code change.

**Pull request merged**